Readcsvpushdb.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+from sqlalchemy import create_engine
+import requests
+
+df = pd.read_csv ('/home/melodie/Téléchargements/archive/COVID-19 Coronavirus.csv')
```

Comment:
I guess you van read it from a dedicated `data` folder and perhaps the file to the PR for reproductibility?